src/components/FormTodo/FormTodo.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { Todo, User } from '../../types';
+
+interface Props {
+  onAdd: (todo: Todo) => void;
+  user: User[];
+  todos: Todo[];
+}
+
+export const FormTodo: React.FC<Props> = ({ onAdd, user, todos }) => {
```

Comment:
The destructured `user` prop is an array, which is correctly used in the component. However, consider renaming it to `users` for clarity and consistency with its type.